### PR TITLE
Update script.sh

### LINKED ISSUE #1: Simplify time calculation logic
In `0.1.0` the time logic is a little redundant. We use `time_start=0` and thus duration is always "some value minus 0" which can be simplified just be saying `duration=$SECONDS`.

### PR DESCRIPTION
**Pull request by viktoriussuwandi**
fix #1

Hi @agardnerIT,
trying to update `duration` variable, hopefully as your expected, please check it out

### CODE REVIEW

**Comment by agardnerIT** [comment]
Thanks for the PR. This isn't quite complete yet. There are variables that can be removed. Please walk through the logic to try and understand why.

If you get stuck, LMK and I'll explain :)

**Comment by viktoriussuwandi** [comment]
Hi @agardnerIT,
already update script.sh by removed time_start variable. Are there any other variables that should be removed ? Please LMK

File: script.sh [hunk 1]
```diff
@@ -16,10 +16,9 @@ then
   HT_SERVICE_NAME=helm
 fi
 
-time_start=0
 $commandToRun
 time_end=$SECONDS
```
Comment (agardnerIT) [inline script.sh]: `time_end` is also now unneccessary. The script sets `time_end` to `$SECONDS` then immediately sets `duration` to `$SECONDS` so `time_end` and `duration` will (almost always) be the same.

Then tracepusher only uses `duration` to `time_end` is never used.

Please remove `time_end` as well.

Comment (viktoriussuwandi) [reply script.sh]: yes, agree with you @agardnerIT, 
now it should be ok, right ?

**Review by agardnerIT — APPROVED** [review]

**Pull request merged**